tangelo/toolboxes/qubit_mappings/mapping_transform.py

```diff
@@ -112,11 +112,11 @@ def fermion_to_qubit_mapping(fermion_operator, mapping, n_spinorbitals=None, n_e
         qubit_operator = jordan_wigner(fermion_operator)
     elif mapping.upper() == "BK":
         if n_spinorbitals is None:
-            raise ValueError("Bravyi Kitaev requires specification of number of qubits.")
+            raise ValueError("Bravyi Kitaev requires specification of number of spin-orbitals.")
```

Comment:
L115-119-130

lets just refactor them to have a single test before ths branches that says

```
if mapping.upper in {"BK", "SCBK", "JKMN"}:
   raise ValueError(f"{mapping.upper()} requires n_spinorbitals to be set"}
```